app/org/sagebionetworks/bridge/play/controllers/UserManagementController.java

```diff
@@ -34,8 +35,12 @@ public Result createUser() throws Exception {
         SignUp signUp = parseJson(request(), SignUp.class);
 
         boolean consent = JsonUtils.asBoolean(node, CONSENT_FIELD);
+        
+        ScheduleContext context = new ScheduleContext.Builder()
+                .withStudyIdentifier(session.getStudyIdentifier())
+                .withClientInfo(getClientInfoFromUserAgentHeader()).build();
```

Comment:
Removed clientInfo entirely from this set of calls. You also can't, at this time, select anything but the default subpopulation when creating a user (as an admin, virtually always to set up a test). I can add this later.